Add new provider : Xing

D4rkTiger opened the pull request:
Add Xing provider to authenticate with german professional social
network account

D4rkTiger committed:
Add new provider : Xing

Add Xing provider to authenticate with german professional social
network account

D4rkTiger committed:
Add DoYouBuzz provider

Add DoYouBuzz provider to authenticate with french CV provider account

D4rkTiger committed:
Revert "Add DoYouBuzz provider"

This reverts commit 95e2e4f2070e7542c7ad1577e04c8f50c11e36ac.

jerriep commented on Owin.Security.Providers/packages.config:
```diff
@@ -1,5 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
+  <package id="Microsoft.Net.Compilers" version="1.1.1" targetFramework="net45" developmentDependency="true" />
```
Why is this package added?

D4rkTiger replied:
It is necessary to compile C# 6.0. Not for you ?

jerriep commented on Owin.Security.Providers/Owin.Security.Providers.csproj:
```diff
@@ -1,5 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="..\packages\Microsoft.Net.Compilers.1.1.1\build\Microsoft.Net.Compilers.props" Condition="Exists('..\packages\Microsoft.Net.Compilers.1.1.1\build\Microsoft.Net.Compilers.props')" />
```
Why did you add this package?

D4rkTiger replied:
It is necessary to compile C# 6.0. Not for you ?

jerriep commented on Owin.Security.Providers/Owin.Security.Providers.csproj:
```diff
@@ -448,8 +464,16 @@
   </ItemGroup>
-  <ItemGroup />
+  <ItemGroup>
+    <Folder Include="DoYouBuzz\" />
```
Why is this empty folder added to the project?

D4rkTiger replied:
Oh sorry, it is a future provider I prepare to provide. I haven't seen that I checked-in

D4rkTiger committed:
Clean project

Remove C# 6 code (VKontakte and Xing)
Remove package allows us to build C#6 code under VS2013
Remove empty provider folder "DoYouBuzz"
